examples/custom-resource/libraries/gordon.rb

```diff
@@ -0,0 +1,15 @@
+class Gordon < Inspec.resource(1)
+  name 'gordon'
+
+  example "
+    describe gordon do
+      its('example') { should eq 'foobar' }
+    end
+  "
+
+  def example
```

Comment:
Likewise, maybe give this a name that makes it clear we're calling a custom resource from another custom resource.

Ooh, you could have a Batman and Bruce Wayne resource, and bruce implements `full_name` (which calls whoami) and Batman implements `secret_identity` (which calls full name on Batman)